shared/src/metabase/shared/util/parameters.cljc

```diff
@@ -0,0 +1,145 @@
+(ns metabase.shared.util.parameters
```

Comment:
it's a bit weird seeing this in `util`, how about `metabase.shared.models.parameters`?
Not ideal but we already have `metabase.shared.models.visualization-settings` so it' less weird.

Reply:
I don't really understand our directory structure for shared code at all, so I agree this is a weird place for it. `models` doesn't really make sense to me though.

Maybe I can make a new `parameters` directory or something.

Reply:
yea, I think parameters directory makes sense.